Resources/Prototypes/_NF/Shipyard/stellaris.yml

```diff
@@ -0,0 +1,30 @@
+- type: vessel
+  id: stellaris
+  name: NT Stellaris
+  description: A mobile theatre perfect for putting on any show.
+  price: 30000
```

Comment:
this is too low, the ship is appraising at 36,000 when purchased. Raise it to 41,000 minimum